frontend/src/components/index.ts

```diff
@@ -1,3 +1,7 @@
 import { ANTVersion } from "./ANTVersion";
+import { Form } from "./Form";
+import { Chain } from "./Chain";
+import { Block } from "./Block";
+import { InitialBlock } from "./InitialBlock";
 
-export { ANTVersion };
+export { ANTVersion, Form, Chain, Block, InitialBlock };
```

Comment:
I don't think we should use this pattern of a single `index.ts` that re-exports all of our components. It effectively breaks code splitting.